.travis.yml

```diff
@@ -1,5 +1,5 @@
 sudo: required
-dist: trusty
+dist: bionic
```

Comment:
here be several angry dragons

Reply:
@rgacogne points out that a new travis would not add much compared to Debian Buster on CircleCI, and that as long as we have Travis, we might as well keep it on the old stuff. This makes sense to me.

Reply:
but, we should still switch that to py3

Reply:
Alright, I'll work on reverting this to Trusty.

Reply:
The latest version of  Python available for Trusty is 3.5, which is EoL, and package maintainers are already adopting 3.6-and-later features in their packages. If we can make this work with Python 3.5 it will have a limited lifetime, as at some point a package we need for building docs will no longer be compatible.

Reply:
I understand -Travis- offers py3.6 in the Trusty image. If that is easy to use, then please try that. If not, we'll spend some more time thinking about what we want with Travis.

Personally I'm tempted to solve the "let's make sure we periodically test on old systems" problem in Github Actions separately.

Reply:
Travis build reverted to Ubuntu Trusty, but with Python 3.6.